src/components/NavbarStudio.scss

```diff
@@ -0,0 +1,158 @@
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
```

Comment:
![image](https://user-images.githubusercontent.com/75971336/200190110-2a48441b-c907-41b6-a4f9-29eafc4e9cb1.png)

Reply:
Yes, but why should I depend on the second style file?

Reply:
done